packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -164,6 +168,40 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     });
   };
 
+  private _getChartDataText(data: IChartProps): string {
+    const chartDataMode = this.props.chartDataMode || ChartDataMode.default;
+    const x = data!.chartData![0].horizontalBarChartdata!.x;
+    const y = data!.chartData![0].horizontalBarChartdata!.y;
+
+    switch (chartDataMode) {
+      case ChartDataMode.default:
+        return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+      case ChartDataMode.fraction:
+        return x + '/' + y;
+      case ChartDataMode.percentage:
+        const dataRatio = Math.round((x / y) * 100);
+        return dataRatio + '%';
+    }
+  }
+
+  private _createBenchmark(data: IChartProps): JSX.Element {
+    if (data.chartData![0].data) {
+      const totalData = data.chartData![0].horizontalBarChartdata!.y;
+      const benchmarkData = data.chartData![0].data;
+      const benchmarkRatio = Math.round(((benchmarkData ? benchmarkData : 0) / totalData) * 100);
+
+      const benchmarkStyles = {
+        marginLeft: 'calc(' + benchmarkRatio + '% - 4px)',
+        marginRight: 'calc(' + (100 - benchmarkRatio) + '% - 4px)'
+      };
+
+      // tslint:disable-next-line:jsx-ban-props
+      return <div className={this._classNames.triangle} style={benchmarkStyles} />;
+    }
+
+    return <></>;
```

Comment:
what is the point of this line?

Reply:
I just want to return an empty element.

Reply:
If it's to return nothing when there's no benchmark, then maybe that logic should be on line 88? Instead of returning an empty object here, don't call the function if there is no benchmark?

Reply:
Yes. I can move this logic to line 88. Add an if statement at line 88.

Reply:
Change it.